src/bundle/Controller/LocationController.php

```diff
@@ -62,4 +64,26 @@ public function actionsAction(
 
         return $this->reloadTab('locations', $content->id, $redirectLocationId);
     }
+
+    public function updateLocationDefaultSortOrderAction(
```

Comment:
As this is in the `Location` controller do we need the keyword `Location` in the action name? I think `updateDefaultSortOrderAction` would be enough. @dpobel  what do you think?